index.bs

```diff
@@ -7155,6 +7155,45 @@ partial dictionary MLOpSupportLimits {
     ::
         The interpolation algorithm used to fill the output tensor values.
 
+        Both algorithms start with these inputs, computed for each spatial axis:
+        ```
+        scale = outputSize / inputSize
```

Comment:
@reillyeon commented offline ".... it's not clear where `inputSize`, `outputSize` and `outputCoordinate` come from" - I will improve this intro text to be explicit about that.

Reply:
Basically
for each axis in relevant axis
    inputSize = input.shape[axis]
endfor

Reply:
Expanded the text in 8a34038